JitsiConference.js

```diff
@@ -91,14 +106,57 @@ function JitsiConference(options) {
     /**
      * Indicates whether the connection is interrupted or not.
      */
-    this.connectionIsInterrupted = false;
+    this.isJvbConnectionIsInterrupted = false;
+
+    /* P2P related fields below: */
+
+    /**
+     * Stores reference to deferred start P2P task. It's created when 3rd
+     * participant leaves the room in order to avoid ping pong effect (it
+     * could be just a page reload).
+     * @type {number|null}
+     */
+    this.deferredStartP2P = null;
+
+    const delay = parseInt(options.config.backToP2PDelay, 10);
+
+    /**
+     * A delay given in seconds, before the conference switches back to P2P
+     * after the 3rd participant has left.
+     * @type {number}
+     */
+    this.backToP2PDelay = isNaN(delay) ? 5 : delay;
+    logger.info(`backToP2PDelay: ${this.backToP2PDelay}`);
+
+    /**
+     * If set to <tt>true</tt> it means the P2P ICE is no longer connected.
+     * When <tt>false</tt> it means that P2P ICE (media) connection is up
+     * and running.
+     * @type {boolean}
+     */
+    this.isP2PConnectionInterrupted = false;
 
+    /**
+     * Flag set to <tt>true</tt> when P2P session has been established
+     * (ICE has been connected).
+     * @type {boolean}
+     */
+    this.p2pEstablished = false;
+
+    /**
+     * A JingleSession for the direct peer to peer connection.
+     * @type {JingleSessionPC}
+     */
+    this.p2pJingleSession = null;
```

Comment:
we never use something like that in our code for class fields, I would find this confusing